rosco-core/src/main/groovy/com/netflix/spinnaker/rosco/providers/aws/config/RoscoAWSConfiguration.groovy

```diff
@@ -63,6 +65,7 @@ class RoscoAWSConfiguration {
     List<AWSVirtualizationSettings> virtualizationSettings = []
   }
 
+  @AutoClone(style = AutoCloneStyle.SIMPLE)
```

Comment:
an alternative to this is `@Immutable(copyWith=true)` and you could do
`virtualizationSettings.copyWith(sourceImage: bakeRequest.base_ami)`